### DISCUSSION TITLE
Brainstorm da Feature Criação de Categoria de itens

### DISCUSSION THREAD

**Discussion by crvzybird** [crvzybird/ess-cardapio#17] [Ideas]
## :bookmark: Bem vindo(a)!
  Essa **_Discussion_**  servirá como um espaço para realizar **_Brainstorms_** da **_Feature_**  **Criação de Categoria de itens**  e qualquer etapa de desenvolvimento dela que esteja na sua faze de conceituação e ideação.Com o intuito de podermos nos manter conectados mesmo não sendo possível estar em reuniões virtuais/presenciais é esperado que comentários sejam feitos para debater possíveis abordagens, implementações e até sugestão de refatorações de implementações que possam contribuir no desenvolvimento dessa *_Feature_**

**Comment by crvzybird** [comment]
@mateusgomes01 @mateussgn

Com a criação das categorias, seria interessante haver um esquema de filtro baseado na categoria escolhida?

Também vocês acham que seria necessário haver uma categoria baseada em "Pratos mais pedidos"?

**Reply by mateussgn** [reply]
A gente pode categorizar por tags.
Dessa forma além de categorias mais funcionais,  podemos ter personalizadas.
